openpype/hosts/blender/plugins/create/create_blendScene.py

```diff
@@ -31,21 +33,36 @@ def _process(self):
         asset = self.data["asset"]
         subset = self.data["subset"]
         name = plugin.asset_name(asset, subset)
-        asset_group = bpy.data.objects.new(name=name, object_data=None)
-        asset_group.empty_display_type = 'SINGLE_ARROW'
-        instances.objects.link(asset_group)
+
+        # Create the new asset group as collection
+        asset_group = bpy.data.collections.new(name=name)
+        instances.children.link(asset_group)
         self.data['task'] = get_current_task_name()
         lib.imprint(asset_group, self.data)
 
-        # Add selected objects to instance
-        if (self.options or {}).get("useSelection"):
-            bpy.context.view_layer.objects.active = asset_group
-            selected = lib.get_selection()
-            for obj in selected:
-                if obj.parent in selected:
-                    obj.select_set(False)
-                    continue
-            selected.append(asset_group)
-            bpy.ops.object.parent_set(keep_transform=True)
+        try:
+            area = next(
+                area for area in bpy.context.window.screen.areas
+                if area.type == 'OUTLINER')
+            region = next(
+                region for region in area.regions
+                if region.type == 'WINDOW')
+        except StopIteration as e:
+            raise RuntimeError("Could not find outliner. An outliner space "
+                               "must be in the main Blender window.") from e
+
+        with bpy.context.temp_override(
+            window=bpy.context.window,
+            area=area,
+            region=region,
+            screen=bpy.context.window.screen
+        ):
+            ids = bpy.context.selected_ids
```

Comment:
I feel like this is best moved to a library function instead, like `get_selected_collections` or whatever which a nice docstring? Or maybe this should be added to `lib.get_selection` but maybe for time being hidden behind a flag `include_collections=True` to avoid issues elsewhere?